packages/timg/build.sh

```diff
@@ -3,15 +3,18 @@ TERMUX_PKG_DESCRIPTION="A terminal image and video viewer"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION="1.6.0"
-TERMUX_PKG_SRCURL=https://github.com/hzeller/timg/archive/24d7727a4ba44b32fcdfb8e21fc258a4638fbbfe.tar.gz
-TERMUX_PKG_SHA256=5fe2a0f68dacfe367cc82ed5c3a21a9c1455ae78df1acedb7c02ed27ef502833
-TERMUX_PKG_AUTO_UPDATE=true
-TERMUX_PKG_DEPENDS="ffmpeg, graphicsmagick, libc++, libjpeg-turbo, libexif, zlib"
+TERMUX_PKG_REVISION="1"
+TERMUX_PKG_SRCURL=https://github.com/hzeller/timg/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=9e1b99b4eaed82297ad2ebbde02e3781775e3bba6d3e298d7598be5f4e1c49af
+TERMUX_PKG_AUTO_UPDATE=trueTERMUX_PKG_DEPENDS="ffmpeg, graphicsmagick, libc++, libcairo, libdeflate, libjpeg-turbo, libexif, librsvg, libsixel, poppler, zlib"
```

Comment:
> `libsixel` not found. Seriously? What kind of error is it?

The kind that happens if you don't put a newline here after.
`TERMUX_PKG_AUTO_UPDATE=true`

Reply:
Ah `TERMUX_PKG_AUTO_UPDATE` isn't linted.
Guess that's another linter PR for me...

Reply:
To be fair, the linter would have caught that if it was any other variable.